From c5a914934e1d1d9449d0e5a5af66112646a8c5e3 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Tue, 6 Aug 2013 11:17:01 +0800
Subject: [PATCH 229/602] ENGR00273962 Update ota server to align with jb4.3
 build server

Update ota server to align with jb4.3 build server

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 imx6/etc/ota.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imx6/etc/ota.conf b/imx6/etc/ota.conf
index f8af05c..a44a432 100644
--- a/imx6/etc/ota.conf
+++ b/imx6/etc/ota.conf
@@ -1,2 +1,2 @@
-server=10.192.224.178
+server=10.192.224.88
 port=10888
-- 
1.8.0

